integration/send-files/e2e/express.spec.ts

```diff
@@ -68,4 +69,44 @@ describe('Express FileSend', () => {
   it('should return an error if the file does not exist', async () => {
     return request(app.getHttpServer()).get('/file/not/exist').expect(400);
   });
+  it('should allow for the client to end the response and be able to make another', async () => {
+    await app.listen(0);
+    const url = await app.getUrl();
+    const [protocol, host, port] = url.replace('[::1]', 'localhost').split(':');
+    const httpOptions = {
+      host: host.replace('//', ''),
+      protocol: `${protocol}:`,
+      port,
+      path: '/file/slow',
+      method: 'GET',
+    };
```

Comment:
I think we could improve the readability of this code by extracting some parts of it to utility methods, maybe defined below the tests. For instance, this block of code, from what I understand, starts the app and prepares the request parameters to be sent. An alternative here could be:
```suggestion
    await app.listen(0);
    const httpOptions =  getHttpOptionsForSlowFileEndpoint()
    };
```